core/controllers/suggestion.py

```diff
@@ -374,3 +374,39 @@ def get(self):
 
         self.values.update({'suggestions': [s.to_dict() for s in suggestions]})
         self.render_json(self.values)
+
+
+class UpdateSuggestionHandler(base.BaseHandler):
+    """"Handles update operations relating to suggestions."""
+
+    # TODO(#11735): Currently, only handles the translation suggestion
+    # updates. This needs to update to generally handle the suggestions.
+    @acl_decorators.can_update_suggestions
+    def put(self, suggestion_id):
+        """Handles PUT requests.
+
+        Raises:
+            InvalidInputException. The suggestion is already handled.
+            InvalidInputException. The 'translation_html' parameter is missing.
+        """
+        suggestion = suggestion_services.get_suggestion_by_id(suggestion_id)
+        if suggestion.is_handled:
+            raise self.InvalidInputException(
+                'The suggestion with id %s has been accepted or rejected'
+                % (suggestion_id)
+            )
+
+        if self.payload.get('translation_html') is None:
+            raise self.InvalidInputException(
+                'The parameter \'translation_html\' is missing.'
+            )
+
+        if self.payload.get('translation_html').isdigit():
```

Comment:
Added a validation for inputs.
Reference: https://github.com/oppia/oppia/pull/12424/files#r612926670
/cc @DubeySandeep

Reply:
Instead of adding `X is unexpected_type_a, unexpected_type_b, ...` check, make it `not X of expected type`.

Reply:
Agreed. Otherwise you miss out on cases where a list, json dict, etc. is entered instead of the expected string.